XTBExtractor test not working on Windows

Some character in the xtb extraction cannot be read.

```================================================================================================= FAILURES =================================================================================================
____________________________________________________________________________________________ test_xtb_extractor ____________________________________________________________________________________________

    def test_xtb_extractor():
        known_output_file = join('../data', 'xtb_energy.output')
    
        # Get properties from output_file.
>       xtbext = stk.XTBExtractor(output_file=known_output_file)

..\test_utilities.py:10:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
..\..\stk\utilities\utilities.py:1281: in __init__
    self.output_lines = f.readlines()
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <encodings.cp1252.IncrementalDecoder object at 0x000002D9C98866A0>
input = b'\n         #    Occupation            Energy/Eh            Energy/eV\r\n      --------------------------------------...C   40 0.952\r\n    45  H   1.000        C   40 0.960\r\n    46  H   0.997
      C   41 0.977\r\n    47  H   0.997   '
final = False

    def decode(self, input, final=False):
>       return codecs.charmap_decode(input,self.errors,decoding_table)[0]
E       UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 5367: character maps to <undefined>

encodings\cp1252.py:23: UnicodeDecodeError
============================================================================ 1 failed, 116 passed, 11 skipped in 22.42 seconds =============================================================================```

--- a/stk/utilities/utilities.py
+++ b/stk/utilities/utilities.py
@@ -1277,7 +1277,8 @@ class XTBExtractor:
 
         """
         self.output_file = output_file
-        # Handle bad default encoding on Windows.
+        # Handle bad default encoding on Windows, which cannot
+        # interpret UTF-8 encoded files by xTB.
         with open(self.output_file, 'r', encoding='UTF-8') as f:
             self.output_lines = f.readlines()
 

--- a/stk/utilities/utilities.py
+++ b/stk/utilities/utilities.py
@@ -1277,6 +1277,7 @@ class XTBExtractor:
 
         """
         self.output_file = output_file
+        # Handle bad default encoding on Windows.
         with open(self.output_file, 'r', encoding='UTF-8') as f:
             self.output_lines = f.readlines()
 

--- a/stk/utilities/utilities.py
+++ b/stk/utilities/utilities.py
@@ -1277,7 +1277,7 @@ class XTBExtractor:
 
         """
         self.output_file = output_file
-        with open(self.output_file, 'r') as f:
+        with open(self.output_file, 'r', encoding='UTF-8') as f:
             self.output_lines = f.readlines()
 
         self._extract_values()

